Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -46,7 +46,6 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
     open func setUp() {
         keyboardDismissMode = .onDrag
         rowHeight = UITableView.automaticDimension
-        estimatedRowHeight = 150
```

Comment:
Have you tried in different iOS versions? I remember this having different effect depending on the UIKit version.

Maybe it is safer to only remove it for certain versions that we managed to check?

Reply:
I've tested with iOS 12 now, and it is working fine 🤔 I also can't find any info about that. Maybe that was in really old versions?

Reply:
@nuno-vieira could we still leverage `estimatedRowHeight` but with some common (awaited row height) -> eg cell with text message (1 liner)?

As per docs:
```
The default value is automaticDimension, which means that the table view selects an estimated height to use on your behalf. Setting the value to 0 disables estimated heights, which causes the table view to request the actual height for each cell. If your table uses self-sizing cells, the value of this property must not be 0.
When using height estimates, the table view actively manages the contentOffset and contentSize properties inherited from its scroll view. Do not attempt to read or modify those properties directly.
```

Reply:
I remember us removing the estimated height 🤔 For short text messages, it was OK but sending/removing large messages (e.g. with image attachments/long text) when content size changes significantly were causing jumps to be more noticeable.

I will give it some testing to see if this is still relevant

Reply:
@bielikb As per the docs, removing the estimatedRowHeight, the estimation is now done automatically by Apple, which it looks it is better than our estimation.